guides/subscriptions.md

```diff
@@ -36,7 +36,7 @@ line:
       {Phoenix.PubSub, name: :my_pubsub},
       # Start the endpoint when the application starts
       MyAppWeb.Endpoint,
-      {Absinthe.Subscription, MyAppWeb.Endpoint}
+      {Absinthe.Subscription, pubsub: MyAppWeb.Endpoint}
```

Comment:
Changing this is a breaking change. We can _add support_ for this style, but the implementation you have here removes support for just passing in the module.

Reply:
The implementation of `child_spec/1` in this PR should maintain compatibility with "just passing  module" - it accepts either an atom or a keyword list:

https://github.com/absinthe-graphql/absinthe/blob/70bd17dcd443cc6b1edee1ce1ca0e74d0ffd5644/lib/absinthe/subscription.ex#L62-L76

However this isn't true for `Absinthe.Subscription.start_link/1` - it now expects a keyword list, but I can change it to also accept just a module if you'd like 🙇